docs/getting-started/concepts.md

```diff
@@ -1,57 +1,67 @@
-1. **private key:** A private key is a series of bits used within an encryption algorithm for encrypting data so that the corresponding ciphertext appears random.
-1. **public evaluation key:** A public evaluation key is used to perform homomorphic operations on encrypted data, typically by a server.
-1. **guaranteed correctness of encrypted computations:** To achieve security, TFHE, the underlying encryption scheme, adds random noise to ciphertexts. This can induce errors during processing of encrypted data, depending on noise parameters. By default, Concrete ML uses parameters that ensure the correctness of the encrypted computation, so there is no need to account for noise parametrization. Therefore, the results on encrypted data will be the same as the results of simulation on clear data.
+-  **Encryption and decryption:** Encryption converts human-readable information (plaintext) into data (ciphertext) that is unreadable by a human or computer unless with the proper key. Encryption takes plaintext and an encryption key and produces ciphertext, while decryption is the reverse operation.
 
-While Concrete ML users only need to understand the cryptography concepts above, for a deeper understanding of the cryptography behind the Concrete stack, please see the [whitepaper on TFHE and Programmable Boostrapping](https://whitepaper.zama.ai/) or [this series of blogs](https://www.zama.ai/post/tfhe-deep-dive-part-1).
 
-## Model accuracy considerations under FHE constraints
+-   **Encrypted inference:** FHE allows third parties to execute a machine learning model on encrypted data. The inference result is also encrypted and can only be decrypted by the key holder.
 
-To respect FHE constraints, all numerical programs that include non-linear operations over encrypted data must have all inputs, constants, and intermediate values represented with integers of a maximum of 16 bits.
+-  **Key generation:** Cryptographic keys are generated using random number generators. Key generation can be time-consuming and produce large keys, but each model used by a client only requires key generation once.
+
+   - **Private encryption key**: A private encryption key is a series of bits used within an encryption algorithm for encrypting data so that the corresponding ciphertext appears random.
+
+   - **Public evaluation key**: A public evaluation key is used to perform homomorphic operations on encrypted data, typically by a server.
+
+-  **Guaranteed correctness of encrypted computations:** To ensure security, TFHE adds random noise to ciphertexts. Depending on the noise parameters, it can cause errors during encrypted data processing. By default, Concrete ML uses parameters that guarantee the correctness of encrypted computations, so the results on encrypted data equals to those of simulations on clear data.
+- **Programmable Boostrapping (PBS)** : TBD
```

Comment:
We need a simple explanation of PBS here as a reference for other docs
cc @andrei-stoian-zama

Reply:
Sure: `programmable bootstrapping enables the homomorphic evaluation of any function of a ciphertext, with a controlled level of noise` (taken from the whitepaper), with a link: https://eprint.iacr.org/2021/091